tools/generator/generator.csproj

```diff
@@ -39,4 +39,15 @@
     <ProjectReference Include="..\..\src\Java.Interop.Tools.JavaCallableWrappers\Java.Interop.Tools.JavaCallableWrappers.csproj" />
   </ItemGroup>
 
+  <!-- Copy package ref symbols for our installers. '$(Pkg*)' props are set during NuGet restore when GeneratePathProperty="true" -->
+  <Target Name="_CopyGeneratorPackageRefPdbsToOutputDirectory"
+      AfterTargets="CopyFilesToOutputDirectory"
+      Inputs="$(PkgHtmlAgilityPack)\lib\Net45\HtmlAgilityPack.pdb"
```

Comment:
Is it worth going ahead and changing this project to `net472` so we can pull in the `netstandard2.0` pdb instead?

Reply:
The symbol file from the path above is identical to the one that was included in a recent .vsix from master. I think if we're targeting net472 we'd still pull in the net45 version of the assembly (as it is the closest match?), though the `HtmlAgilityPack.dll` from the .vsix I checked wasn't a binary match with any of the versions in the HtmlAgilityPack NuGet installation directory...